packages/deploy-tooling/test/unit/cli/index.test.ts

```diff
@@ -49,13 +64,6 @@ describe('cli', () => {
             errorMock.mockClear();
         });
 
-        test('missing mandatory parameter --config', () => {
```

Comment:
Could you please add a test case with a command containing all required parameters if the `--config` is not provided? 
I would like to have such a test case for documentation purposes.

Reply:
Thanks for adding to https://github.com/SAP/open-ux-tools/pull/929/files